.changeset/shiny-dingos-remember.md

```diff
@@ -0,0 +1,21 @@
+---
+"astro-remote": major
+---
+
+This Update will bring all Dependencies to current versions as well as resolve some typing issues with new versions
+
+Breaking:
+- Markdown sanitation has been disabled due to complete removal of all custom data (Hope to resolve soon, but the current workaround is functioning)
+
+Commit history:
+
+- Initial Prep: Repo organization and Prepare for upgrade
+- More Cleanup: Code Cleanup before I started Upgrading
+- Some Progress: First Commit to include upgrade progress, mostly upgrading of Deps before any testing and fixing
+- Code Cleanup: Initial cleanup of code errors
+- Working Tests: Time to work out bugs and get things working
+- Test Cleanup: Tests are working and most previous features have been fixed.
+- More Cleanup: remove all deps used for testing different results and their code
+- Fix Inital Type Errors left behind and lint!
+- Readme update: updated readme now that all the errors are gone.
+- small fix to Utils file
```

Comment:
Similar comment here, _we_ can already browse the commit history on this PR! This context is helpful for my code review, but it isn't actionable for users of `astro-remote`.

This section can be removed entirely.

```suggestion
```